build.gradle

```diff
@@ -235,6 +235,22 @@ task copyConnectorToConfluent(type: Copy, group: confluentTestingGroup, dependsO
   into "${confluentHome}/share/confluent-hub-components"
 }
 
+// Installs the connector using defaults for parameters. This means:
+//  1) It uses $CONFLUENT_HOME to determine which CP installation to use
+//  2) It automatically uninstalls any current version of the connector
+//  3) It automatically adds the "confluent-hub-components" directory to all known worker properties files
+task installConnectorInConfluent(type: Exec, group: confluentTestingGroup, dependsOn: [connectorArchive]) {
+  description = "Uses 'Confluent-hub' to install the connector in your local Confluent Platform"
+  commandLine "confluent-hub", "install", "--no-prompt", "build/distro/${baseArchiveName}.zip"
+  ignoreExitValue = true
+}
+
+task installDatagenConnectorInConfluent(type: Exec, group: confluentTestingGroup, dependsOn: [connectorArchive]) {
```

Comment:
This is a one-time task that's already documented in the CONTRIB file - what value does a Gradle task add for it?